src/BlockTypes/ProductCategories.php

```diff
@@ -77,9 +86,11 @@ protected function render( $attributes, $content ) {
 			}
 		}
 
-		$classes = $this->get_container_classes( $attributes );
+		$classes_and_styles = $this->get_container_classes( $attributes );
+		$classes            = $classes_and_styles[0];
+		$styles             = $classes_and_styles[1];
 
-		$output  = '<div class="' . esc_attr( $classes ) . '">';
+		$output  = '<div class="' . esc_attr( $classes ) . '" style="' . $styles . '">';
```

Comment:
We should escape every variable added to the output:
```suggestion
		$output  = '<div class="' . esc_attr( $classes ) . '" style="' . esc_attr( $styles ) . '">';
```